fix.js

```diff
@@ -32,20 +32,20 @@ function Server(opt, func) {
      this.server = net.createServer(function(stream) {
 
         stream.setTimeout(2 * 60 * 1000);//if no traffic for two minutes, kill connection!
-        var session = this;
```

Comment:
This was causing multiple clients to override each other session as 'this' is always pointing to the server object.